### ISSUE TITLE
SSD Mobilenet V1 FPN (shared weights) 640x640 -published results are not reproduceable

### ISSUE THREAD

**Issue by tamirhailo** [tensorflow/models#7861]
### System information
- **What is the top-level directory of the model you are using**: tensorflow/models/research/object_detection
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: Yes - used the given pipeline.config file, and updated the batch size to 16 instead of 128
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 16.04.5 LTS (GNU/Linux 4.4.0-1098-aws x86_64v)
- **TensorFlow installed from (source or binary)**: Binary 
- **TensorFlow version (use command below)**: 1.12
- **Bazel version (if compiling from source)**: 
- **CUDA/cuDNN version**: 9.0
- **GPU model and memory**: Tesla V100-SXM2 16152MB
- **Exact command to reproduce**: 
    python object_detection/model_main.py \
        --pipeline_config_path=${pipeline_path} \
        --model_dir=${MODEL_DIR} \
        --num_train_steps=${NUM_TRAIN_STEPS} \
        --sample_1_of_n_eval_examples=$SAMPLE_1_OF_N_EVAL_EXAMPLES \
        --alsologtostderr

### Describe the problem
I've tried to reproduce the results published in model zoo for the following model:
*ssd_mobilenet_v1_fpn_shared_box_predictor_640x640_coco14_sync_2018_07_03*
I've used the original pipeline which is given with the checkpoint.
$pipeline_path  = the pipeline config that is given in the downloaded model from the model zoo - batch size was updated to 16.
${NUM_TRAIN_STEPS} = 12500 

Given the differences between the pipeline.config batch size (which was 128) and mine, which was 16, i've tried to run more scenarios where i the Learning Rate by 1/8 and x8 the number of steps.
and still didn't reproduce the 33%  published.

I tried tweaking the hyperparameters, but the maximum mAP I got on the COCO validation set was 30% while using the original pre-trained checkpoint for evaluation gave 35%.

I really want to be able to reproduce the results, 
what am i missing? what should I do? is it possible that you will share the exact settings needed for reproducing?

### Source code / logs
the updated part of the LR, and number of iterations. I tried it after the original config file didn't reproduced the results
![Screenshot from 2019-11-24 15-09-48](https://user-images.githubusercontent.com/52370932/69495159-807cc800-0ecc-11ea-860f-e400a0ce7836.png)

**Comment by pkulzc** [comment]
Batch size is the key. We used TPU to do synchronized training so effective batch size is 128 as in the config. Larger batch size gives you more stable gradient.

You can also use 8 GPUs to do synchronized training with the same config, it should give you similar results.
